packages/xchain-client/src/types.ts

```diff
@@ -124,7 +124,7 @@ export interface XChainClient {
   getFees(options?: FeeEstimateOptions): Promise<Fees>
 
   transfer(params: TxParams): Promise<TxHash>
-  // prepareTx(params: TxParams): Promise<UnsignedTx>
+  prepareTx(params: TxParams): Promise<string>
```

Comment:
needs a package bump & change log filled in 
`yarn updatePackages patch Update "change log here " client ` 
then run 
`yarn updateDeps <version> client `